biogrid/apps/biogrid-mvp/src/app/components/simulation-board.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { Distance, ItemPosition } from '@biogrid/grid-simulator';
+import { ReactComponent as SmallBatterySvg } from '../../assets/icons/battery-small.svg';
+import { ReactComponent as LargeBatterySvg } from '../../assets/icons/battery-large.svg';
+import { ReactComponent as HouseSvg } from '../../assets/icons/house.svg';
+import { ReactComponent as GridSvg } from '../../assets/icons/grid.svg';
+import { ReactComponent as SolarPanelSvg } from '../../assets/icons/solar-panel.svg';
+import './simulate-board.css';
+
+export interface GridItemRet {
+  gridItemName: string;
+  relativePosition: ItemPosition;
+}
+
+export interface GridItemLines {
+  fromItem: string;
+  toItem: string;
+}
+
+interface SimBoardProps {
+  height: Distance;
+  width: Distance;
+  items: GridItemRet[];
+  lines: GridItemLines[];
+}
+
+interface ItemsByName {
+  [name: string]: ItemPosition;
+}
+
+const Grid = () => {
+  return (
+    <div className="grid-item">
+      <GridSvg />
+    </div>
+  );
+};
+const LargeBattery = () => {
```

Comment:
when these elements end up in the same position and overlap, it's hard to tell what's happening in the UI
Can we make elements smaller and display them next to each other instead of having one on top of the other?

Reply:
Oh, this will get fixed in a future PR from the backend, let me comment that in

Reply:
Even if this is fixed in BE, in general, it's good practice to have a way to handle this in FE as well, though fine to leave as a TODO with a ticket